frontend/packages/kubevirt-plugin/src/components/modals/vm-flavor-modal/vm-flavor-modal.tsx

```diff
@@ -43,7 +43,7 @@ import { getVMTemplateNamespacedName } from '../../../selectors/vm-template/sele
 
 const getId = (field: string) => `vm-flavor-modal-${field}`;
 
-const getFlavors = (template: TemplateKind) => {
+const getAvailableFlavors = (template: TemplateKind) => {
```

Comment:
piggybacking this here
@yaacov is this name better?

Reply:
:+1: as long as it's different than the global one